### PR TITLE
updates to docs

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
Pull Request
**Are you a current Fivetran customer?** 

Internal
**What change(s) does this PR introduce?** 

as part of effort to standardize and update package docs
**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [x] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)
No, but a lot of facelift warrants a refresh
**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [x] No 

**How did you test the PR changes?** 

 

- [x] CircleCi  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Comment by fivetran-reneeli** [comment]
I noticed since the package hasn't been updated recently, the `generate columns` [macro](https://github.com/fivetran/dbt_salesforce_source/tree/main/macros) is condensed into one file. Assuming we want to break this out too

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-reneeli thanks for applying these updates! I have a few requests to change before approving that are both in the comment lines and listed out below. Let me know if you have any questions!

- [ ] I didn't mention this in the checklist (but am adding it now) that we want to be explicit with our jinja conditionals to make them easier to read. Would you mind updating the passthrough conditionals in the models to follow the format [here](https://github.com/fivetran/dbt_zendesk_source/blob/e41038ab6fc369aa9f9f01f802f981c9bb582abf/models/stg_zendesk__ticket.sql#L25-L30)
- [x] However, can you be sure to adjust the whitespace from the above changes as outlined [here](https://github.com/fivetran/dbt_style_guide/blob/updates/joe-additions/dbt_package_guidelines.md#jinja-formatting)
- [x] Add the `identifier` to each of the sources and include a variable for users to overwrite. See [here](https://github.com/fivetran/dbt_apple_store_source/blob/333d6fc5049f7579b04932993aa1effe8b4d323c/models/src_apple_store.yml#L10-L19) for an example.

File: README v2.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    salesforce_schema: your_schema_name 
+```
+### Disabling Components
+Your Salesforce connector might not sync every table that this package expects. If you do not have the `SPRINT`, `COMPONENT`, or `VERSION` tables synced, add the following variable to your root `dbt_project.yml` file:
+
+```yml
+vars:
+    salesforce_using_sprints: false   # Disable if you do not have the sprint table, or if you do not want sprint related metrics reported
+    salesforce_using_components: false # Disable if you do not have the component table, or if you do not want component related metrics reported
+    salesforce_using_versions: false # Disable if you do not have the versions table, or if you do not want versions related metrics reported
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply README v2.md:62]: My bad, didn't catch this, thank you!

File: README v2.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
+<p align="center">
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply README v2.md:1]: Yes that's the plan, sorry if there was any confusion! Holding off so I can just move stuff over. I am going to replace the old README once we get this one finalized.

File: models/stg_salesforce__opportunity.sql [hunk 3]
```diff
@@ -75,9 +67,10 @@ with source as (
 
         {% endif %}
 
-    from macro
+    from fields
```
Comment (fivetran-joemarkiewicz) [inline models/stg_salesforce__opportunity.sql:70]: Can you adjust the formatting of the jinja conditional in this model

File: models/stg_salesforce__user.sql [hunk 4]
```diff
@@ -66,18 +56,17 @@ with source as (
         user_role_id,
         user_type,
         username 
-      --The below script allows for pass through columns.
+        --The below script allows for pass through columns.
```
Comment (fivetran-joemarkiewicz) [inline models/stg_salesforce__user.sql:59]: Jinja conditional needs to be adjusted.

File: README v2.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+### Database and Schema Variables
+By default, this package will run using your target database and the `salesforce` schema. If this is not where your Salesforce data is (perhaps your Salesforce schema is `salesforce_fivetran`), add the following configuration to your root `dbt_project.yml` file:
+
+```yml
+vars:
+    salesforce_database: your_database_name
+    salesforce_schema: your_schema_name 
+
+    # if an individual source table has a different name than expected, provide the name of the table as it appears in your warehouse
+    <default_source_table_name>_identifier: your_table_name
```
Comment (fivetran-joemarkiewicz) [inline README v2.md:56]: Can you adjust this to be `salesforce_<default_source_table_name>_identifier`

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-reneeli I have a few change requests before this will be good to go! Let me know if you have any questions.

File: models/tmp/stg_salesforce__account_tmp.sql [hunk 6]
```diff
@@ -1,5 +1,5 @@
 select *
 from {{ var('account') }}
-{% if var('using_account_history_mode_active_records', false) %}
+{% if var('using_opportunity_history_mode_active_records', false) %}
```
Comment (fivetran-joemarkiewicz) [inline models/tmp/stg_salesforce__account_tmp.sql:3]: Why was this changed to opportunity? Can you change this back to `using_account_history_mode_active_records`

**Review by fivetran-joemarkiewicz — APPROVED** [review]
I have one question and a few requests to update the jinja formatting. Once you make those changes this will be good to merge to the release branch.

**Pull request merged**